<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serveur</class>
 <widget class="QWidget" name="Serveur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serveur</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>781</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QTableView" name="tableViewDataBase">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditPort"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Port :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QPushButton" name="pushButtonConnexion">
      <property name="text">
       <string>Connexion</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBoxDatabase">
      <item>
       <property name="text">
        <string>Choisir les donn√©es</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Personnages</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Score</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Parties</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
